app/src/main/java/br/org/cesar/discordtime/stickysessions/data/repository/mapper/SessionMapper.java

```diff
@@ -32,6 +33,9 @@ public Session mapToDomain(SessionRemote dataType) {
         Session session = new Session();
         session.id = dataType.getId();
         session.topics = dataType.getTopics();
+        session.title = dataType.getTitle();
+        session.description = dataType.getDescription();
```

Comment:
Maybe do the check here.
`session.description = dataType.getDescription != null ? dataType.getDescription : defaultValue`

Same thing for the color.